[core-kit] app-arch/pigz Autogen package app-arch/pigz (parallel gzip implementation)

org-tekeli-borisp opened the pull request:
Introduce a new package for pigz, a parallel implementation of gzip, with initial templates and metadata. This includes the ebuild file and autogen.yaml for automated package generation. Removed from gentoo staging list

(cherry picked from commit 220ee60d7e4da832bbed05217face56cbbe19bb7) (cherry picked from commit 59d0bb144cea21a0a4f9d3ab168e1a54e73d834c)

org-tekeli-borisp committed:
[core-kit] app-arch/pigz Autogen package app-arch/pigz (parallel gzip implementation)

Introduce a new package for pigz, a parallel implementation of gzip, with initial templates and metadata. This includes the ebuild file and autogen.yaml for automated package generation. Removed from gentoo staging list

(cherry picked from commit 220ee60d7e4da832bbed05217face56cbbe19bb7)
(cherry picked from commit 59d0bb144cea21a0a4f9d3ab168e1a54e73d834c)

cuantar commented:
I'd made an issue for this at the Funtoo Jira.  Glad to see it implemented.

cuantar commented on core-kit/next/app-arch/pigz/autogen.yaml:
next -> mark

org-tekeli-borisp replied:
Ah sorry! Due to cherry peeking!

cuantar commented on core-kit/next/app-arch/pigz/templates/pigz.tmpl:
next -> mark

geaaru replied:
Yeah, exactly. Thanks for the review

cuantar reviewed (changes requested):
The template and autogen YAML should be in the `mark` release subdir, rather than `next`, right?

org-tekeli-borisp committed:
[core-kit]  app-arch/pigz Relocate pigz metadata from 'next' to 'mark'.

Moved autogen.yaml and template files for app-arch/pigz to the 'mark' directory from 'next'. This change is purely organizational with no modifications to the file content.

cuantar reviewed (approved):
Thanks :)
